src/test/java/org/broadinstitute/hellbender/engine/dataflow/RefAPISourceUnitTest.java

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.hellbender.engine.dataflow;
+
+import com.google.cloud.dataflow.sdk.Pipeline;
+import com.google.cloud.dataflow.sdk.options.PipelineOptionsFactory;
+import com.google.cloud.dataflow.sdk.testing.TestPipeline;
+import com.google.cloud.genomics.dataflow.utils.GenomicsOptions;
+import org.broadinstitute.hellbender.engine.dataflow.datasources.RefAPIMetadata;
+import org.broadinstitute.hellbender.engine.dataflow.datasources.RefAPISource;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.reference.ReferenceBases;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+public class RefAPISourceUnitTest extends BaseTest {
+
+    private ReferenceBases queryReferenceAPI( final String referenceName, final SimpleInterval interval ) {
+        GenomicsOptions options = PipelineOptionsFactory.create().as(GenomicsOptions.class);
+        options.setApiKey(getDataflowTestApiKey());
+        options.setProject(getDataflowTestProject());
+
+        final Pipeline p = TestPipeline.create(options);
+
+        Map<String, String> referenceNameToIdTable = RefAPISource.buildReferenceNameToIdTable(p.getOptions(), referenceName);
+        RefAPISource refAPISource = new RefAPISource();
+        RefAPIMetadata refAPIMetadata = new RefAPIMetadata(referenceName, referenceNameToIdTable);
+
+        return refAPISource.getReferenceBases(p.getOptions(), refAPIMetadata, interval);
+    }
+
+    @Test(groups = "cloud")
+    public void testReferenceSourceQuery() {
+        final ReferenceBases bases = queryReferenceAPI("EOSt9JOVhp3jkwE", new SimpleInterval("1", 50000, 50009));
```

Comment:
I think it may have been GRCh37, but I can't quite remember. Switching to human-readable reference names is a ticket (https://github.com/broadinstitute/hellbender/issues/646).